src/pages/_app.tsx

```diff
@@ -110,7 +112,9 @@ function App({ Component, pageProps: { session, ...pageProps } }: AppProps) {
       <PostHogProvider client={posthog}>
         <SessionProvider session={session}>
           <ChakraProvider theme={extendThemeWithNextFonts}>
-            <MyApp Component={Component} pageProps={pageProps} />
+            <TooltipProvider>
```

Comment:
would not want TooltipProvider here ideally
+do we really need tiptap's styles here, considering that we only use it in the sponsor dashboard

we should be very very apprehensive of anything we are planning to add in the _app.tsx folder

Reply:
this isnt tiptap's tooltip

this is shadcn's tooltip, [https://ui.shadcn.com/docs/components/tooltip](https://ui.shadcn.com/docs/components/tooltip)

this only needs to be initialised once, and not everywhere we use shadcn's tooltip

Reply:
and the css, nextjs now allowing to import .css file anywhere other than _app.tsx

tried making it moduled, but some or the other thing kept breaking in description. lets keep this for now.

Reply:
i know the tooltip is shadcn's
lets remove it from _app.tsx

look for more ways to remove the CSS file from _app.tsx as well

Reply:
moved tooltip provider to its instances.

but make sure not to use multiple TooltipProviders in a single page.